cves/CVE-2002-0661.yml

```diff
@@ -107,8 +120,8 @@ subsystem:
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
-  answer:
-  name:
+  answer: Based on filepaths and bug report description.
```

Comment:
May want to give a little more detail here, like a path to the corresponding directory or something. Not 100% necessary, but this answer struck me as a bit sparse

Reply:
Alright, I'll elaborate on the filepath

Reply:
Cheers. That'll add a nice little bit of context for how you came up with the subsystem name